### ISSUE TITLE
suggestion: export position

### ISSUE THREAD

**Issue by Wirless** [OTAcademy/RME#12]
![image](https://github.com/OTAcademy/RME/assets/25297250/a70be10a-b97c-4952-bfa9-bb0075a60780)
need to have export positionns when creating some shit for arena and stuff will save everyone million times 
e.g annihilator tiles

  participantsPos = {
        Position(4786, 5199, 8), -- Player 1, this player should be the one to pull the lever
        Position(3118, 1842, 7), -- Player 2
        Position(3119, 1842, 7), -- Player 3
        Position(3120, 1842, 7) -- Player 4
    },

**Comment by Dip-Set1** [comment]
To clarify, are you saying you want to be able to highlight an area and copy position of all tiles highlighted?

**Comment by Wirless** [comment]
yes basically. for smoother lever creation for quests and such. maybe its time to add IDE into RME or something idk.. would be quite handy with some script menu and stuff.

**Comment by Wirless** [comment]
taking on this for OTARMEIE this iwll be 
map_display.cpp file changes mainly where copy position is shown for single selection add option for multi selection
.
